airbyte-cdk/python/airbyte_cdk/utils/print_buffer.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+
+import sys
+import time
+from io import StringIO
+from threading import RLock
+from types import TracebackType
+from typing import Optional
+
+
+class PrintBuffer:
+    def __init__(self, flush_interval: float = 0.1):
```

Comment:
Have you experimented with different values of flush timer? Do we get better boost if we flush every 0.5 or 1 second?

Reply:
tried to increase the interval, nothing changed much.